bc_obps/compliance/service/penalty_calculation_service.py

```diff
@@ -342,6 +349,25 @@ def calculate_penalty(
 
         return result
 
+    @classmethod
+    def create_initial_penalties_for_obligation(
+        cls,
+        obligation: ComplianceObligation,
+    ) -> None:
+        """Create empty automatic overdue and late submission penalties with NONE status for a new obligation."""
+
+        CompliancePenalty.objects.create(
+            compliance_obligation=obligation,
+            penalty_type=CompliancePenalty.PenaltyType.AUTOMATIC_OVERDUE,
+            status=CompliancePenalty.Status.NONE,
+        )
+
+        CompliancePenalty.objects.create(
+            compliance_obligation=obligation,
+            penalty_type=CompliancePenalty.PenaltyType.LATE_SUBMISSION,
+            status=CompliancePenalty.Status.NONE,
+        )
+
```

Comment:
Let's chat about the approach here. I don't think we need to be creating penalty records for every obligation. The calculate_penalty functions with persist=False should be enough to show an ephemeral penalty as of a date.